cal/cal/static/css/calendaranalytics.scss

```diff
@@ -25,6 +25,12 @@ button {
   padding: 0 5 5 5;
 }
 
+.textbox input {
```

Comment:
Instead of .textbox input you can just place the textbox class on the input element itself. We also namespace all of our custom classnames with `cal-foo-bar` to avoid name collisions with bootstrap and other libraries, and to make it obvious that we're not using bootstrap styling (so it should be `cal-textbox`). There are other examples of this in this same scss file.